ci/jenkins/jobs/macros.yaml

```diff
@@ -231,9 +231,10 @@
 
           make clean
           docker pull antrea/openvswitch --all-tags
-          docker images | grep 'antrea-ubuntu' | awk '{print $3}' | xargs -r docker rmi || true
-          # Clean up dangling images generated in previous builds. Recent ones must be excluded
-          # because they might be being used in other builds running simultaneously.
+          docker images | grep "${JOB_NAME}" | awk '{print $3}' | xargs -r docker rmi -f || true
+          docker images | grep '<none>' | awk '{print $3}' | xargs -r docker rmi || true
```

Comment:
I think `docker image prune -f --filter "until=1h" || true` will remove the `<none>` images. Some `<none>` images are used by other jobs during build.

Reply:
But `docker image prune` fails to delete some `<none>` images after a long time.

Reply:
Could you check why "docker image prune" didn't delete those ones? because of errors or they were referenced? If the latter, then the new command might not work either?

Reply:
I rechecked images on the node and those "<none>" images can be cleaned up with current code.